engine/gui/src/gui.cpp

```diff
@@ -395,6 +395,8 @@ namespace dmGui
         scene->m_DestroyRenderConstantsCallback = params->m_DestroyRenderConstantsCallback;
         scene->m_CloneRenderConstantsCallback = params->m_CloneRenderConstantsCallback;
         scene->m_OnWindowResizeCallback = params->m_OnWindowResizeCallback;
+        scene->m_ApplyLayoutCallback    = params->m_ApplyLayoutCallback;
```

Comment:
Why are all of these called something with Callback? Seems strange... Shouldn't it be ApplyLayoutFn, SetMaterialPropertyFn etc?